core/azure-core-amqp/pom.xml

```diff
@@ -39,7 +39,7 @@
     <url>https://github.com/Azure/azure-sdk-for-java</url>
     <connection>scm:git:https://github.com/Azure/azure-sdk-for-java.git</connection>
     <developerConnection>scm:git:https://github.com/Azure/azure-sdk-for-java.git</developerConnection>
-    <tag>v1.0.0-preview.1</tag>
+    <tag>v1.0.0-preview.2</tag>
```

Comment:
I don't expect this tag is adding anything meaningful it is just duplication that can only get out of date. We should remove it here and in the other pom file that sets the tag to the version.

Reply:
Removed the tag